x-pack/plugins/xpack_main/server/routes/api/v2/telemetry/telemetry.js

```diff
@@ -91,9 +97,21 @@ export function telemetryRoute(server) {
       const config = req.server.config();
       const start = req.payload.timeRange.min;
       const end = req.payload.timeRange.max;
+      const unencrypted = req.payload.unencrypted;
 
       try {
-        return await getTelemetry(req, config, start, end);
+        if (unencrypted) {
```

Comment:
does `unencrypted === false` means: give me unencrypted data? what do you think to name the flag `asEncrypted`